recipes/leftbar/unread_chip/unread_chip.vue

```diff
@@ -65,23 +60,27 @@ export default {
 
 <style lang="less">
 .dt-leftbar-unread-chip {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: var(--space-200) var(--space-500) var(--space-200) var(--space-400);
+  gap: var(--space-300);
+  font-size: var(--fs-100);
+  box-shadow: var(--bs-md);
+  border-radius: var(--br-pill);
+  border: none;
+  height: 28px;
```

Comment:
Magic number alert 🤓

One core principle in authoring CSS is to avoid setting a _fixed_ height at all costs, with some unavoidable exceptions. We almost always want the combination of font and box-model properties to dictate the minimum height and let the flow naturally occur.

Solution is most likely to add `line-height: var(--lh-600);` and remove `height`.